homeassistant/components/blebox/climate.py

```diff
@@ -0,0 +1,91 @@
+"""BleBox climate entity."""
+
+from homeassistant.components.climate import ClimateDevice
```

Comment:
The `ClimateDevice` name is legacy now. Swap it for `ClimateEntity`.

Reply:
Fixed in: https://github.com/home-assistant/core/pull/36143